projects/plugins/classic-theme-helper-plugin/src/class-classic-theme-helper-plugin.php

```diff
@@ -22,6 +22,29 @@ class Classic_Theme_Helper_Plugin {
 	public function __construct() {
 	}
 
+	/**
+	 * Initialize the plugin.
+	 */
+	public static function init() {
+
+		// Init Jetpack packages
+		add_action( 'plugins_loaded', array( static::class, 'init_packages' ), 1 );
+	}
+
+	/**
+	 * Configure what Jetpack packages should get automatically initialized.
+	 *
+	 * @return void
+	 */
+	public static function init_packages() {
+		if ( class_exists( 'Automattic\Jetpack\Classic_Theme_Helper\Main' ) ) {
```

Comment:
The `class_exists` addition, while a better idea anyway incase there are issues with packages, was prompted by a failing test. More in this job: https://github.com/Automattic/jetpack/actions/runs/9805786819/job/27076220128